releasenotes/notes/0.24/add-clifford-from-matrix-3184822cc559e0b7.yaml

```
@@ -1,9 +1,11 @@
 ---
-features:
+features_quantum_info:
   - |
-    Added :meth:`.Clifford.from_matrix` and :meth:`.Clifford.from_operator` method that
-    creates a ``Clifford`` object from its unitary matrix and operator representation respectively.
+    Added two new constructor methods, :meth:`.Clifford.from_matrix` and
+    :meth:`.Clifford.from_operator`, that create a :class:`~.Clifford` object
+    from its unitary matrix and operator representation respectively.
   - |
-    The constructor of :class:`.Clifford` now can take any Clifford gate object up to 3 qubits
-    as long it supports :meth:`to_matrix` method,
-    including parameterized gates such as ``Rz(pi/2)``, which were not convertible before.
+    The constructor of :class:`.Clifford` now can take any Clifford gate object
+    up to 3 qubits as long it supports :meth:`to_matrix` method,
```

Since `to_matrix` refers to the method on _any_ object, the `:meth:` qualifier shouldn't be here, should it?
```suggestion
    up to 3 qubits as long it implements a ``to_matrix`` method,
```